mvc/views/tag-helpers/intro.rst

```diff
@@ -1,8 +1,281 @@
+
+.. literalinclude:: authoring/sample/AuthoringTagHelpers/src/AuthoringTagHelpers/Views/_ViewImportsCopy.cshtml
+   :language: html
+   :emphasize-lines: 2
+   :lines: 1-2
+
+The ``@addTagHelper`` directive makes Tag Helpers available to the view. In this case, the view file is *Views/_ViewImports.cshtml*, which by default is included in all views, which means that the Tag Helpers will be available to all views. The code above uses the wildcard syntax ("*") to specify that all Tag Helpers in the specified assembly (*Microsoft.AspNet.Mvc.TagHelpers*) will be available. The first parameter after ``@addTagHelper`` specifies the Tag Helpers to load (we are using "\*" for all Tag Helpers), and the second parameter "Microsoft.AspNet.Mvc.TagHelpers" specifies the assembly containing the Tag Helpers. *Microsoft.AspNet.Mvc.TagHelpers* is the assembly for the built-in ASP.NET 5 Tag Helpers.
+
+To expose all of the Tag Helpers in this project (which creates an assembly named *AuthoringTagHelpers*), you would use the following:
+
+.. literalinclude:: authoring/sample/AuthoringTagHelpers/src/AuthoringTagHelpers/Views/_ViewImportsCopy.cshtml
+   :language: html
+   :emphasize-lines: 3
+
+If your project contains an ``EmailTagHelper`` with the default namespace (``AuthoringTagHelpers.TagHelpers.EmailTagHelper``), you can provide the fully qualified name (FQN) of the Tag Helper:
+
+.. FQN syntax
+
+.. literalinclude:: authoring/sample/AuthoringTagHelpers/src/AuthoringTagHelpers/Views/_ViewImports.cshtml
+   :language: html
+   :lines: 1-3
+   :emphasize-lines: 3
+
+To add a Tag Helper to a view using an FQN, you first add the FQN (``AuthoringTagHelpers.TagHelpers.EmailTagHelper``), and then the assembly name (*AuthoringTagHelpers*). Most developers prefer to use the  "\*" wildcard syntax. The wildcard syntax allows you to insert the wildcard character "\*" as the suffix in an FQN. For example, any of the following directives will bring in the ``EmailTagHelper``:
+
+.. code-block:: c#
+
+	@addTagHelper "AuthoringTagHelpers.TagHelpers.E*, AuthoringTagHelpers"
+	@addTagHelper "AuthoringTagHelpers.TagHelpers.Email*, AuthoringTagHelpers"
+
+As mentioned previously, adding the ``@addTagHelper`` directive to the *Views/_ViewImports.cshtml* file makes the Tag Helper available to all views. You can use the ``@addTagHelper`` directive in specific view files if you want to opt-in to exposing the Tag Helper to only those views.
+
+``@removeTagHelper`` removes Tag Helpers
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The ``@removeTagHelper`` has the same two parameters as ``@addTagHelper``, and it removes a Tag Helper that was previously added. For example, ``@removeTagHelper`` applied to a specific view removes the specified Tag Helper from the view. Using ``@removeTagHelper`` in a *Views/Folder/_ViewImports.cshtml* file removes the specified Tag Helper from all of the views in *Folder*.
```

Comment:
Hmmm I like how you say: `removes the specified Tag Helper from all of the views in *Folder*`. Maybe instead of saying adjacent/sub-directory above you can do something similar.